### PR TITLE
Replace <see langword="Single" /> tags

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml [hunk 1]
```diff
@@ -1571,9 +1571,9 @@
         <Parameter Name="Value" Type="System.Single" />
       </Parameters>
       <Docs>
-        <param name="Value">The <see langword="Single" /> value to convert.</param>
+        <param name="Value">The <see langword="T:System.Single" /> value to convert.</param>
         <summary>Converts a <see cref="T:System.Single" /> value (a single-precision floating point number) to a <see cref="T:System.String" /> value.</summary>
-        <returns>The <see langword="String" /> representation of the <see langword="Single" /> value.</returns>
+        <returns>The <see langword="String" /> representation of the <see langword="T:System.Single" /> value.</returns>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml]:
```suggestion
        <returns>The string representation of the <see langword="T:System.Single" /> value.</returns>
```

File: xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml [hunk 2]
```diff
@@ -1748,10 +1748,10 @@
         <Parameter Name="NumberFormat" Type="System.Globalization.NumberFormatInfo" Index="1" FrameworkAlternate="netcore-3.0;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="Value">The <see langword="Single" /> value to convert.</param>
+        <param name="Value">The <see langword="T:System.Single" /> value to convert.</param>
         <param name="NumberFormat">The number format to use, according to <see cref="T:System.Globalization.NumberFormatInfo" />.</param>
         <summary>Converts a <see cref="T:System.Single" /> value to a <see cref="T:System.String" /> value, using the specified number format.</summary>
-        <returns>The <see langword="String" /> representation of the <see langword="Single" /> value.</returns>
+        <returns>The <see langword="String" /> representation of the <see langword="T:System.Single" /> value.</returns>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml]:
```suggestion
        <returns>The string representation of the <see langword="T:System.Single" /> value.</returns>
```

File: xml/Microsoft.VisualBasic.CompilerServices/StringType.xml [hunk 3]
```diff
@@ -601,7 +601,7 @@
         <AssemblyVersion>10.0.0.0</AssemblyVersion>
       </AssemblyInfo>
       <Docs>
-        <summary>Returns a <see langword="String" /> value that corresponds to a specified <see langword="Single" /> and optional number format information.</summary>
+        <summary>Returns a <see langword="String" /> value that corresponds to a specified <see langword="T:System.Single" /> and optional number format information.</summary>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic.CompilerServices/StringType.xml]:
```suggestion
        <summary>Returns a string that corresponds to a specified <see langword="T:System.Single" /> and optional number format information.</summary>
```

File: xml/Microsoft.VisualBasic.CompilerServices/StringType.xml [hunk 4]
```diff
@@ -645,8 +645,8 @@
         <Parameter Name="Value" Type="System.Single" />
       </Parameters>
       <Docs>
-        <param name="Value">Required. <see langword="Single" /> to convert to a <see langword="String" /> value.</param>
-        <summary>Returns a <see langword="String" /> value that corresponds to a specified <see langword="Single" />.</summary>
+        <param name="Value">Required. <see langword="T:System.Single" /> to convert to a <see langword="String" /> value.</param>
+        <summary>Returns a <see langword="String" /> value that corresponds to a specified <see langword="T:System.Single" />.</summary>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic.CompilerServices/StringType.xml]:
```suggestion
        <summary>Returns a string that corresponds to a specified <see langword="T:System.Single" />.</summary>
```

File: xml/Microsoft.VisualBasic.CompilerServices/StringType.xml [hunk 5]
```diff
@@ -688,9 +688,9 @@
         <Parameter Name="NumberFormat" Type="System.Globalization.NumberFormatInfo" />
       </Parameters>
       <Docs>
-        <param name="Value">Required. <see langword="Single" /> to convert to a <see langword="String" /> value.</param>
+        <param name="Value">Required. <see langword="T:System.Single" /> to convert to a <see langword="String" /> value.</param>
         <param name="NumberFormat">A <see cref="T:System.Globalization.NumberFormatInfo" /> object that defines how numeric values are formatted and displayed, depending on the culture.</param>
-        <summary>Returns a <see langword="String" /> value that corresponds to a specified <see langword="Single" /> and number format information.</summary>
+        <summary>Returns a <see langword="String" /> value that corresponds to a specified <see langword="T:System.Single" /> and number format information.</summary>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic.CompilerServices/StringType.xml]:
```suggestion
        <summary>Returns a string that corresponds to a specified <see langword="T:System.Single" /> and number format information.</summary>
```

File: xml/Microsoft.VisualBasic.Compatibility.VB6/Support.xml [hunk 6]
```diff
@@ -631,7 +631,7 @@
       </Parameters>
       <Docs>
         <param name="CurrentFont">A <see cref="T:System.Drawing.Font" />.</param>
-        <param name="Size">A <see langword="Single" /> that represents the Visual Basic 6.0 <see langword="FontSize" /> property.</param>
+        <param name="Size">A <see langword="T:System.Single" /> that represents the Visual Basic 6.0 <see langword="FontSize" /> property.</param>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic.Compatibility.VB6/Support.xml]: For types, `cref` should be used, not `langword`. The latter is reserved for keywords like `true`, `false` or `null`:
```suggestion
        <param name="Size">A <see cref="T:System.Single" /> that represents the Visual Basic 6.0 <see langword="FontSize" /> property.</param>
```

File: xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml [hunk 7]
```diff
@@ -1160,7 +1160,7 @@
       <Docs>
         <param name="Value">The object to convert.</param>
         <summary>Converts an object to a <see cref="T:System.Single" /> value.</summary>
-        <returns>The <see langword="Single" /> value of the object.</returns>
+        <returns>The <see langword="T:System.Single" /> value of the object.</returns>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml]:
```suggestion
        <returns>The <see cref="T:System.Single" /> value of the object.</returns>
```

File: xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml [hunk 8]
```diff
@@ -1197,7 +1197,7 @@
       <Docs>
         <param name="Value">The string to convert.</param>
         <summary>Converts a <see cref="T:System.String" /> to a <see cref="T:System.Single" /> value.</summary>
-        <returns>The <see langword="Single" /> value of the string.</returns>
+        <returns>The <see langword="T:System.Single" /> value of the string.</returns>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml]:
```suggestion
        <returns>The <see cref="T:System.Single" /> value of the string.</returns>
```

File: xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml [hunk 9]
```diff
@@ -1571,9 +1571,9 @@
         <Parameter Name="Value" Type="System.Single" />
       </Parameters>
       <Docs>
-        <param name="Value">The <see langword="Single" /> value to convert.</param>
+        <param name="Value">The <see langword="T:System.Single" /> value to convert.</param>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml]: Please apply the same change to the rest of the files:

```suggestion
        <param name="Value">The <see cref="T:System.Single" /> value to convert.</param>
```

**Comment by NextTurn** [comment]
> `langword` should not be used for types, it should be `cref`. I offered a few suggested changes, but please apply the same change to the rest of them.

Sorry, I simply misspelled the regex this time.

> `<returns>` tags don't need to specify the type of the return value, since it will already show up next to the string in MS Docs, so you can use natural language.

I've made changes to the places mentioned in your comments.

File: xml/System.Windows.Automation.Provider/ITextProvider.xml [hunk 10]
```diff
@@ -235,7 +235,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets a value that specifies whether a text provider supports selection and, if so, the type of selection supported.</summary>
-        <value>One of <see langword="None" />, <see langword="Single" />, or <see langword="Multiple" /> from <see cref="T:System.Windows.Automation.SupportedTextSelection" />.</value>
+        <value>One of <see langword="None" />, <see cref="T:System.Single" />, or <see langword="Multiple" /> from <see cref="T:System.Windows.Automation.SupportedTextSelection" />.</value>
         <remarks>To be added.</remarks>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation.Provider/ITextProvider.xml]: I'm not convinced the `Single` mentioned here is referring to the `T:System.Single`.

File: xml/System.Windows.Automation/TextPattern.xml [hunk 11]
```diff
@@ -1601,7 +1601,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets a value that specifies whether a text provider supports selection and, if so, the type of selection supported.</summary>
-        <value>One of <see langword="None" />, <see langword="Single" />, or <see langword="Multiple" /> from <see cref="T:System.Windows.Automation.SupportedTextSelection" />.</value>
+        <value>One of <see langword="None" />, <see cref="T:System.Single" />, or <see langword="Multiple" /> from <see cref="T:System.Windows.Automation.SupportedTextSelection" />.</value>
         <remarks>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/TextPattern.xml]: Same here. I think this is something different. Can you please confirm?

File: xml/System.Xaml/XamlLanguage.xml [hunk 12]
```diff
@@ -989,8 +989,8 @@
         <ReturnType>System.Xaml.XamlType</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a <see cref="T:System.Xaml.XamlType" /> for the <see langword="Single" /> XAML language intrinsic.</summary>
-        <value>A <see cref="T:System.Xaml.XamlType" /> for the <see langword="Single" /> XAML language intrinsic.</value>
+        <summary>Gets a <see cref="T:System.Xaml.XamlType" /> for the <see cref="T:System.Single" /> XAML language intrinsic.</summary>
+        <value>A <see cref="T:System.Xaml.XamlType" /> for the <see cref="T:System.Single" /> XAML language intrinsic.</value>
```
Comment (carlossanlop) [inline xml/System.Xaml/XamlLanguage.xml]: I think these two may also ve referring to something other than `T:System.Single`.

File: xml/System.Web.UI.WebControls/ListBox.xml [hunk 13]
```diff
@@ -594,7 +594,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the selection mode of the <see cref="T:System.Web.UI.WebControls.ListBox" /> control.</summary>
-        <value>One of the <see cref="T:System.Web.UI.WebControls.ListSelectionMode" /> values. The default value is <see langword="Single" />.</value>
+        <value>One of the <see cref="T:System.Web.UI.WebControls.ListSelectionMode" /> values. The default value is <see cref="T:System.Single" />.</value>
```
Comment (carlossanlop) [inline xml/System.Web.UI.WebControls/ListBox.xml]: This one seems to be referring to an enum value. It should also not be using langword, but the API in the cref needs to be different.

File: xml/Microsoft.VisualBasic/VariantType.xml [hunk 14]
```diff
@@ -505,7 +505,7 @@
       <MemberValue>4</MemberValue>
       <Docs>
         <summary>
-          <see langword="Single" />. (-3.402823E+38 through -1.401298E-45 for negative values; 1.401298E-45 through 3.402823E+38 for positive values.) This member is equivalent to the Visual Basic constant <see langword="vbSingle" />.</summary>
+          <see cref="T:System.Single" />. (-3.402823E+38 through -1.401298E-45 for negative values; 1.401298E-45 through 3.402823E+38 for positive values.) This member is equivalent to the Visual Basic constant <see langword="vbSingle" />.</summary>
```
Comment (carlossanlop) [inline xml/Microsoft.VisualBasic/VariantType.xml]: All the values in this enum type seem to be describing another type and should also be fixed. For example, in line 480 of this same file, langword=Short" should be cref=T:System.Short".

File: xml/System.Windows.Automation.Provider/ITextProvider.xml (hunk 10 above)
Comment (NextTurn) [reply xml/System.Windows.Automation.Provider/ITextProvider.xml]: Updated. I don't think it's necessary to list all possible enum values.

File: xml/System.Xaml/XamlLanguage.xml (hunk 12 above)
Comment (NextTurn) [reply xml/System.Xaml/XamlLanguage.xml]: There are referring to the `x:Single` type in XAML, which maps to `System.Single` in .NET but they are different things. I'm changing these to `<code>`.

File: xml/Microsoft.VisualBasic/VariantType.xml (hunk 14 above)
Comment (NextTurn) [reply xml/Microsoft.VisualBasic/VariantType.xml]: These seem to be referring to VB types rather than .NET types. Can you confirm?

Comment (carlossanlop) [reply xml/Microsoft.VisualBasic/VariantType.xml]: The enum summary is saying these are types, but I'm seeing also Null. I don't know if that one is treated as a type in VB or a keyword.
I say we exclude this file from the PR. Can you please revert it?

Comment (carlossanlop) [reply xml/Microsoft.VisualBasic/VariantType.xml]: If you can confirm that the APIs described in each enum's descriptions using langword are actually types, feel free to change them to crefs.

Comment (NextTurn) [reply xml/Microsoft.VisualBasic/VariantType.xml]: Reverted. These are actually VB keywords, although they don't show up in `<see langword="" />` completion list.

**Pull request merged**